priv/repo/migrations/20240723145806_create_detours.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Skate.Repo.Migrations.CreateDetours do
+  use Ecto.Migration
+
+  @disable_ddl_transaction true
+  @disable_migration_lock true
```

Comment:
question: are these only needed due to creating the index in the same migration as the table?

_If so_, I think we should split these up into two migrations, so that we're not disabling transactions for the table creation.

Reply:
Oh I seeeeeeeee. No, these are not only needed due to creating the index in the same migration as the table: disabling transactions was [recommended](https://mbtace.sentry.io/issues/5644599926/?project=5303878&query=is%3Aunresolved+issue.priority%3A%5Bhigh%2C+medium%5D&referrer=issue-stream&statsPeriod=14d&stream_index=0) for creating indexes concurrently at all.

But! Your comment points out the best answer to [this question](https://mbta.slack.com/archives/C047ZT8ACP5/p1721835486727689?thread_ts=1721834111.986619&cid=C047ZT8ACP5), and that is we should definitely separate these creations (so that we don't disable transactions for the table, as you note)

Reply:
They're split out now ✅